EpikCord/__init__.py

```diff
@@ -434,7 +434,7 @@ def event(self, func):
 
     async def ready(self, data: dict):
         self.user: ClientUser = ClientUser(self.session, data.get("user"))
-        application_response = await self.session.get("https://discord.com/api/v9/oauth2/applications/@me", headers={"Authorization": f"Bot {self.token}"})
+        application_response = await self.session.get("https://discord.com/api/v10/oauth2/applications/@me", headers={"Authorization": f"Bot {self.token}"})
```

Comment:
![image](https://user-images.githubusercontent.com/73362400/154052528-78bd03f7-21d6-4294-b058-a20237e664e1.png)
V10 isn't documented *yet*, I'd like to add it once it's the recommended one version to use.

Reply:
K